### ISSUE TITLE
Array field in inline admin: losing data on first save

### ISSUE THREAD

**Issue by Tonkonozhenko** [unfoldadmin/django-unfold#808]
**What version of Unfold are you using?**

For example: 0.40.0

**What version of Django are you using?**

For example: 5.1.2

**What browser are you using?**

For example: Chrome

**Did you check changelog/commit history, if the bug is not already fixed?**

For example: Yes

**Did you search other issues, if the bug is not already fixed?**

For example: Yes

**Did you check the documentation?**

For example: Yes

**Are you able to replicate the bug in the demo site?**

Didn't find an example.

**Describe your issue**

_Code:_
```python
class Child:
    array_field: list[float] = ArrayField(models.FloatField(), null=True, blank=True)

class ChildInlineAdmin():
    model = Child

@admin.register(Parent)
class ParentAdmin():
    inlines = [ChildInlineAdmin]
```

_Steps to reproduce:_
* I add a new child to the parent
* I add a new array_field item to the child
* I save the parent

_Issue:_ only the first array_field item is saved, all others are lost.

I managed to fix the problem, by adding the next snippet to [inline.js updateElementIndex()](https://github.com/django/django/blob/main/django/contrib/admin/static/admin/js/inlines.js#L25).

```javascript
            if (el.tagName === 'TEMPLATE') {
                const fragment = $(el.content);
                fragment.find("*").each(function() {
                    updateElementIndex(this, prefix, ndx);
                });
            }
```

But I needed to copy the whole file to my app. I wanted to create a PR to unfold (or probably django), but honestly I don't know what is the best solution here.

**Comment by lukasvinclav** [comment]
Thanks for report. Fixed in #836

**Comment by Tonkonozhenko** [comment]
thank you @lukasvinclav
